x-pack/legacy/plugins/lens/public/app_plugin/plugin.tsx

```diff
@@ -109,6 +115,12 @@ export class AppPlugin {
 
 const app = new AppPlugin();
 
-export const appSetup = () => app.setup(npSetup.core);
-export const appStart = () => app.start(npStart.core);
+export const appSetup = () =>
+  app.setup(npSetup.core, {
+    data: dataSetup,
+  });
+export const appStart = () =>
+  app.start(npStart.core, {
+    data: dataStart,
```

Comment:
@lizozom I'm passing in the instance of `dataStart` which currently matches the type of `DataSetup`